pkgs/top-level/python-packages.nix

```diff
@@ -10167,8 +10167,6 @@ self: super: with self; {
 
   oath = callPackage ../development/python-modules/oath { };
 
-  oauth = callPackage ../development/python-modules/oauth { };
-
   oauth2 = callPackage ../development/python-modules/oauth2 { };
```

Comment:
This one also implements OAuth 1.0.

Reply:
So, should I do something?
Should I maybe add a recommendation in the `throw` message to "use any of the many alternatives"

Reply:
I was thinking about also removing that one, but maybe it's not old enough for that.

Reply:
Oh, I see.
I guess we could remove it too and just revert it if anyone complains (which I doubt).
Also, the name is just confusing, so maybe better for the package search that way.